### PR TITLE
Remove default exports (#261)

### PR DESCRIPTION
**Pull request by warpech**
Implements #261

### CODE REVIEW

**Comment by warpech** [comment]
The Vue demo at https://github.com/Palindrom/palindrom.github.io/blob/ffc1db8411f0b6d33f12ac8d6ccf50129988da6b/public/lab/vue/src/App.vue#L9 is already compatible with this change, because it uses a named export.

File: README.md [hunk 1]
```diff
@@ -59,14 +59,20 @@ npm install palindrom --save
...
-//CommonJS
-var Palindrom = require('palindrom');
+// CommonJS
+const Palindrom = require('palindrom').Palindrom;
+const PalindromDOM = require('palindrom').PalindromDOM;
 
 // ES6/TS
-import Palindrom from 'palindrom'
+import { Palindrom } from 'palindrom';
+import { PalindromDOM } from 'palindrom';
```
Comment (tomalec) [inline README.md]: Why not `import { Palindrom, PalindromDOM} from 'palindrom';`?

File: README.md [hunk 2]
```diff
@@ -59,14 +59,20 @@ npm install palindrom --save
...
 ```
 
-###### Or just download it manually from [github](https://github.com/Palindrom/Palindrom/archive/master.zip).
+Note: The NPM package uses an entry point without a default export. Depending on your environment (Node or browser), you should choose one of the two named exports available: `Palindrom` and `PalindromDOM`.
```
Comment (tomalec) [inline README.md]: I'd say "you could" as I see reasonable use cases to use both in browser.

File: README.md (hunk 1 above)
Comment (warpech) [reply README.md]: In my mind these are two different examples, all you ever need is either Palindrom or PalindromDOM. Should I present it better?

Comment (tomalec) [reply README.md]: no, it's fine as it is, that was just a question.

File: README.md (hunk 2 above)
Comment (warpech) [reply README.md]: I deleted this whole sentence :)

**Comment by warpech** [comment]
I made few changes that I think make it very clear which named export to use. I think that the lack of default export does not require explanation.

**Pull request merged**